plugins/CoreAdminHome/Menu.php

```diff
@@ -47,9 +47,26 @@ public function configureAdminMenu(MenuAdmin $menu)
     public function configureTopMenu(MenuTop $menu)
     {
         $url = $this->urlForModuleAction('CoreAdminHome', 'home');
-
         $menu->registerMenuIcon('CoreAdminHome_Administration', 'icon-settings');
         $menu->addItem('CoreAdminHome_Administration', null, $url, 980, Piwik::translate('CoreAdminHome_Administration'));
+
+        $model = new UsersModel();
+        $user = $model->getUser(Piwik::getCurrentUserLogin());
+        if ($user) {
+            $userChanges = new UserChanges($user);
+
+            $newChangesStatus = $userChanges->getNewChangesStatus();
+            if (!Piwik::isUserIsAnonymous() && Piwik::isUserHasSomeViewAccess() && $newChangesStatus !== UserChanges::NO_CHANGES_EXIST) {
```

Comment:
I guess it would make sense to move the check for `!Piwik::isUserIsAnonymous() && Piwik::isUserHasSomeViewAccess()` around the new code, so fetching the changes isn't even tried if it's not needed.